listenbrainz_spark/recommendations/create_dataframes.py

```diff
@@ -37,7 +37,7 @@ def save_dataframe_html(users_df_time, recordings_df_time, playcounts_df_time, t
     }
     save_html(queries_html, context, 'queries.html')
 
-def create_training_data_from_window():
+def get_listens_for_train_model_window():
```

Comment:
get_listens_for_training_model_window would be more idiomatic english.

Reply:
And you two the same function name here and in the utils module -- they differ and they should have different names, no?